.github/workflows/ci_macos.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         echo "BUILD_MODE=$BUILD_MODE" >> $GITHUB_ENV
 
         DO_PUBLISH='false'
-        if [ "${{ github.event.inputs.publish }}" == "on" ]; then 
+        if [[ "${{ github.event.inputs.publish }}" == "on" || "$BUILD_MODE" == "nightly_build" ]]; then 
```

Comment:
Probably doesn't matter, but this construct requires bash

Reply:
On MacOS by default bash as shell

Reply:
Yes, I know. Just mentioning it, it is not a POSIX shell feature.